README.md

```diff
@@ -18,7 +18,7 @@ _Collaborate and work together on GitHub._
   Do not use quotes on the <details> tag attributes.
 -->
 
-<!--step0-->
+<!--step0
```

Comment:
I wouldn't make a comment on this area. Doesn't it include important information?